package.json

```diff
@@ -18,13 +18,19 @@
     "client:dev": "yarn --cwd packages/client dev",
     "db:migrate": "yarn migration:run",
     "deploy": "git push heroku master",
+    "dev": "yarn --cwd packages/old-accounter dev",
+    "gateway:dev": "yarn --cwd packages/gateway dev",
+    "gateway:watch": "yarn --cwd packages/gateway watch",
     "generate": "concurrently -c blue,green -n GraphQL,SQL \"yarn generate:graphql\" \"yarn generate:sql\"",
+    "generate-subgraph": "cd packages/server && yarn build-subgraph-schema",
```

Comment:
use command with `:` on server to make it callable from everywhere in the monorepo (just `yarn *:*`)
and avoid directory change command

Reply:
same with `--cwd` commands